src/wallet/wallet.h

```diff
@@ -1222,6 +1224,7 @@ class CWallet final : public CCryptoKeyStore, private interfaces::Chain::Notific
 
     /** Add a KeyOriginInfo to the wallet */
     bool AddKeyOrigin(const CPubKey& pubkey, const KeyOriginInfo& info);
+    bool AddKeyOriginWithDB(WalletBatch& batch, const CPubKey& pubkey, const KeyOriginInfo& info);
```

Comment:
nit: May be a bad example to follow, but `AddKeypoolPubkeyWithDB` has `batch` as the last argument, which makes as much sense to me.

Reply:
I was following the `AddKeyPubKeyWithDB` way of argument ordering.